modules/ossm-migrating-network-policies-security.adoc

```diff
@@ -0,0 +1,231 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-network-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-network-policies-security_{context}"]
+= Migrating network policies for security reasons
+
+This procedure applies only if you must recreate your network policies first for security reasons.
```

Comment:
Maybe something like this would be more clear: "This procedure applies only if you want network policies in place _during_ your migration."